MonkeyChat/ConversationListViewController.swift

```diff
@@ -657,54 +599,36 @@ extension ConversationsListViewController {
       return
     }
     
+    // save message
     DBManager.store(message)
     
     //check if conversation is already created
     let conversationId = message.conversationId(Monkey.sharedInstance().monkeyId())
     var conversation = self.conversationHash[conversationId]
```

Comment:
can we change this to `var conversation = self.conversationHash[conversationId] ?? DBManager.getConversation(conversationId)`

so we can reduce to one if in lines 608, 609, 610